sdks/python/apache_beam/yaml/examples/aggregation_using_sqlstatement.yaml

```
@@ -0,0 +1,59 @@
+# coding=utf-8
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This examples reads from a public file stored on Google Cloud. This
+# requires authenticating with Google Cloud, or setting the file in
+#`ReadFromText` to a local file.
+#
+# To set up Application Default Credentials,
+# see https://cloud.google.com/docs/authentication/external/set-up-adc for more
+# information
+#
+# The following example reads mock transaction data from resources/products.csv
+# then performs a simple filter for "Electronics".
+pipeline:
+  transforms:
+    - type: ReadFromCsv
+      name: ReadInputFile
+      config:
+        path: gs://apache-beam-samples/beam-yaml-blog/products.csv
+        delimiter: ","
+    - type: MapToFields
+      name: MapToFields
+      input: ReadInputFile
+      config:
+        language: sql
+        fields:
+          transaction_id: "transaction_id"
+          product_name: "product_name"
+          category: "category"
+          prices: "price"
+    - type: Sql
+      name: SqlCommand 
+      config:
+        query: "select category, count(*) as cnt,sum(prices) as sum_prices, max(prices) as max_prices from PCOLLECTION group by category"
```

Let's use more descriptive column names that describe the intention
```suggestion
        query: "select category, count(*) as num_sold, sum(prices) as revenue, max(prices) as max_price from PCOLLECTION group by category"
```